lte/gateway/python/magma/pipelined/ebpf/ebpf_dl_handler.c

```diff
@@ -26,6 +26,8 @@ struct dl_map_key {
 struct dl_map_info {
   u32 remote_ipv4;
   u32 tunnel_id;
+  u64 bytes;
+  u8 user_data[16];
 };
```

Comment:
We need to define this struct in single header file, its already diverged compared to OAI header definition.

Reply:
yeah I'm exploring this option